### DISCUSSION TITLE
River Instead Of XMonad

### DISCUSSION THREAD

**Discussion by LoganKavalSchool** [LoganKaval/MulBX#154] [Ideas]
We have been developing configuration files for River and the necessary applications. Over the last few weeks my team and I have been working on the tiling functions of the window manager. River seems to be stuck in a tiling only mode and I am trying to fix it. I decided to change the window manager for many reasons. One of the main reasons was because of the protocol each support. Xorg is slowly aging and has some unwanted problems. Wayland on the other hand has problems too, but a bigger community to fix them. I believe that Wayland will over throw Xorg. Therefore we have change to a Wayland based window manager, River.

**Comment by LoganKaval** [comment]
It is Official. The Next Release will have River Instead of XMonad.

**Comment by LoganKaval** [comment]
Since Debian GNU/Linux Does Not Have River In Their Repos, the Debian Version Will Still Have XMonad.
